documentation/modules/deploying/con-deploy-cluster-operator-from-operator-hub.adoc

```diff
@@ -1,21 +1,26 @@
 // Module included in the following assemblies:
 //
-// deploying/assembly_deploy-cluster-operator.adoc
+// deploying.adoc
 
-[id='deploying-cluster-operator-from-operator-hub-{context}']
-= Deploying the Cluster Operator from OperatorHub.io
+[id='deploying-strimzi-from-operator-hub-{context}']
+= Deploying Strimzi from OperatorHub.io
 
 [role="_abstract"]
 {OperatorHub} is a catalog of Kubernetes Operators sourced from multiple providers.
-It offers you an alternative way to install stable versions of Strimzi using the Strimzi Kafka Operator.
+It offers you an alternative way to install stable versions of Strimzi using the *Strimzi Kafka* operator.
 
-The {OperatorLifecycleManager} is used for the installation and management of all Operators published on {OperatorHub}.
+The {OperatorLifecycleManager} is used for the installation and management of all Operators published on OperatorHub.io.
```

Comment:
Do we really want to capitalise _Operators_ here? It does not make much sense to me, especially since  _Strimzi Kafka operator_ below uses lowercase `o`.

Reply:
No. Lower case for generic. Fixed.